docs/service-brokers/docs/012-configuration-helm-broker-bundles.md

```diff
@@ -91,6 +98,18 @@ The `create-instance-schema.json` file contains a schema used to define the para
 This file is not required.
 For more information about the content of the `create-instance-schema.json` file, see the [Input parameters](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#input-parameters-schema-object) specification.
 
+#### The bind-instance-schema.json file
+
+The `bind-instance-schema.json` file contains a schema used to define the parameters. Each input parameter is expressed as a property within a JSON object.
+This file is not required.
+For more information about the content of the `bind-instance-schema.json` file, see the [Input parameters](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#input-parameters-schema-object) specification.
+
+#### The update-instance-schema.json file
+
+The `update-instance-schema.json` file contains a schema used to define the parameters. Each input parameter is expressed as a property within a JSON object.
```

Comment:
Ther eis no any comment it is about an update (or the binding in the previous) operation

Reply:
I developed each sentence to mention an operation to which it applies.